source/configuration_manual/stats/index.rst

```diff
@@ -56,6 +56,7 @@ Group by
 
 .. dovecotadded:: 2.3.10 adds support for implicit discrete aggregation
 .. dovecotchanged:: 2.3.11 adds support for explicit aggregation functions
+.. dovecotchanged:: 3.0.0,2.4.0 allows sub-metric names up to 256 bytes in total, before it was 32 per label
```

Comment:
This might be indented to fit into the 80 char columns, but not necessary. Otherwise OK!

Reply:
hmm. i wonder if i can do that kind of indentation.

Reply:
better?